doc/modules/sgd.rst

```diff
@@ -289,6 +293,7 @@ supports averaged SGD. Averaging can be enabled by setting ``average=True``.
 .. rubric:: Examples
 
 - :ref:`sphx_glr_auto_examples_linear_model_plot_sgdocsvm_vs_ocsvm.py`
+- :ref:`sphx_glr_auto_examples_applications_plot_prediction_latency.py`
```

Comment:
The [Prediction Latency example](https://scikit-learn.org/stable/auto_examples/applications/plot_prediction_latency.html) does not use `sklearn.linear_model.SGDOneClassSVM`
```suggestion
```